scapy/automaton.py

```diff
@@ -1313,17 +1314,19 @@ def run(self,
             elif c.type == _ATMT_Command.BREAKPOINT:
                 raise self.Breakpoint("breakpoint triggered on state [%s]" % c.state.state, state=c.state.state)  # noqa: E501
             elif c.type == _ATMT_Command.EXCEPTION:
-                six.reraise(c.exc_info[0], c.exc_info[1], c.exc_info[2])
+                value = c.exc_info[0]() if c.exc_info[1] is None else c.exc_info[1]  # type: ignore  # noqa: E501
```

Comment:
This snippet is kinda obscure. Maybe add a comment?

Reply:
Truth is... I did not really understand the code ;-)

Reply:
That's the best I could do for now... I propose that we merge it to avoid conflicts (since that's the exact same code than before) and investigate for a better comment (and possibly cleaner code) later. What do you think?

Reply:
Sure. Just to be sure what do you mean "before"? (where did you get that snippet from? ^^')